Doc/whatsnew/3.13.rst

```diff
@@ -492,138 +493,123 @@ Incremental garbage collection
+     class C[T]:
+         type Alias = lambda: T
 
-* :mod:`multiprocessing`, :mod:`concurrent.futures`, :mod:`compileall`:
-  Replace :func:`os.cpu_count` with :func:`os.process_cpu_count` to select the
-  default number of worker threads and processes. Get the CPU affinity
-  if supported.
-  (Contributed by Victor Stinner in :gh:`109649`.)
+  (Contributed by Jelle Zijlstra in :gh:`109118` and :gh:`118160`.)
 
-* :func:`os.path.realpath` now resolves MS-DOS style file names even if
-  the file is not accessible.
-  (Contributed by Moonsik Park in :gh:`82367`.)
+* :ref:`Future statements <future>` are no longer triggered by
+  relative imports of the :mod:`__future__` module,
+  meaning that statements of the form  ``from .__future__ import ...``
+  are now simply standard relative imports, with no special features activated.
+  (Contributed by Jeremiah Gabriel Pascual in :gh:`118216`.)
 
-* Fixed a bug where a :keyword:`global` declaration in an :keyword:`except` block
-  is rejected when the global is used in the :keyword:`else` block.
+* :keyword:`global` declarations are now permitted in :keyword:`except` blocks
+  when that global is used in the :keyword:`else` block.
+  Previously this raised an erroneous :exc:`SyntaxError`.
   (Contributed by Irit Katriel in :gh:`111123`.)
 
-* Many functions now emit a warning if a boolean value is passed as
-  a file descriptor argument.
-  This can help catch some errors earlier.
-  (Contributed by Serhiy Storchaka in :gh:`82626`.)
-
-* Added a new environment variable :envvar:`PYTHON_FROZEN_MODULES`. It
-  determines whether or not frozen modules are ignored by the import machinery,
-  equivalent of the :option:`-X frozen_modules <-X>` command-line option.
+* Add a new environment variable :envvar:`PYTHON_FROZEN_MODULES`.
+  which determines whether frozen modules are ignored by the import machinery,
+  equivalent to the :option:`-X frozen_modules <-X>` command-line option.
   (Contributed by Yilei Yang in :gh:`111374`.)
 
-* Add :ref:`support for the perf profiler <perf_profiling>` working without
-  frame pointers through the new environment variable
-  :envvar:`PYTHON_PERF_JIT_SUPPORT` and command-line option :option:`-X perf_jit
-  <-X>` (Contributed by Pablo Galindo in :gh:`118518`.)
+* Add :ref:`support for the perf profiler <perf_profiling>` working
+  without `frame pointers <https://en.wikipedia.org/wiki/Call_stack>`_ through
+  the new environment variable :envvar:`PYTHON_PERF_JIT_SUPPORT`
+  and command-line option :option:`-X perf_jit <-X>`
+  (Contributed by Pablo Galindo in :gh:`118518`.)
+
+* The location of a :file:`.python_history` file can be changed via the
+  new :envvar:`PYTHON_HISTORY` environment variable.
+  (Contributed by Levi Sabah, Zackery Spytz and Hugo van Kemenade
+  in :gh:`73965`.)
+
+* Classes have a new :attr:`~class.__static_attributes__` attribute.
+  This is populated by the compiler with a tuple of the class' attribute names
```

Comment:
```suggestion
  This is populated by the compiler with a tuple of the class's attribute names
```

Guided by pronunciation.

Reply:
Yeah, I wasn't sure about this one. Simpler would be change to `type's`, but less accurate!